pkg/virtualKubelet/translation/serviceEnv/serviceEnv.go

```diff
@@ -0,0 +1,120 @@
+package serviceEnv
+
+import (
+	apimgmgt "github.com/liqotech/liqo/pkg/virtualKubelet/apiReflection"
+	"github.com/liqotech/liqo/pkg/virtualKubelet/storage"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	v1helper "k8s.io/kubernetes/pkg/apis/core/v1/helper"
+	"k8s.io/kubernetes/pkg/kubelet/envvars"
+)
+
+var masterServices = sets.NewString("kubernetes")
+
+func TranslateServiceEnvVariables(pod *v1.Pod, localNS string, nattedNS string, cacheManager storage.CacheManagerReader) (*v1.Pod, error) {
+	resPod := pod.DeepCopy()
```

Comment:
This `deepcopy` is not needed, we already work on a brand-new pod obtained from [here](https://github.com/liqotech/liqo/blob/734d7544da18c8af58d9aae44df37167965b80a0/pkg/virtualKubelet/translation/mutation.go#L105)